.github/workflows/interchaintest.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+          password: ${{ secrets.GHCR_PAT }}
```

Comment:
@rjonczy are you ok with that?

Reply:
Github does not allow me to add a secret named GITHUB_ so I have to change its name.

Reply:
yeah, i understand. non necessary it is your job may be even to do so. may be Composable SRE should help.

from my side I can generate and plug whatever token you need in this repo if needed (via terraform).

Reply:
but I am ok in general with custom token. so i know that using custom tokens breaks more often than it should